<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medicineview</class>
 <widget class="QWidget" name="medicineview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>科室管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="placeholderText">
            <string>请输入姓名进行查找</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSearch">
           <property name="text">
            <string>查找</string>
           </property>
           <property name="icon">
            <iconset resource="rsc.qrc">
             <normaloff>:/img/search.svg</normaloff>:/img/search.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btAdd">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset resource="rsc.qrc">
             <normaloff>:/img/add.svg</normaloff>:/img/add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btDelete">
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset resource="rsc.qrc">
             <normaloff>:/img/delete.svg</normaloff>:/img/delete.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btEdit">
           <property name="text">
            <string>修改</string>
           </property>
           <property name="icon">
            <iconset resource="rsc.qrc">
             <normaloff>:/img/modify.svg</normaloff>:/img/modify.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSortAscending">
           <property name="text">
            <string>升序</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSortAscending2">
           <property name="text">
            <string>降序</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btExport">
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btImport">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
